### DISCUSSION TITLE
Remove border from Modal

### DISCUSSION THREAD

**Discussion by brandongregoryscott** [twilio-labs/paste#3022] [Q&A] [answered]
I'm looking to build out a custom modal that can render a full-width image (with bleed) in the header, primarily for feature announcements in the Segment application, like this:

<img width="626" alt="image" src="https://user-images.githubusercontent.com/11774799/217294385-5ae49a96-dc75-49ea-8e9b-cd9f348b6652.png">

It looks like the [`Modal`](https://paste.twilio.design/components/modal) component includes a 1px border by default, so I can't get the edge-to-edge bleed I want. Is there an easy way to remove this style, or do I need to use the [`ModalDialogPrimitive`](https://paste.twilio.design/primitives/modal-dialog-primitive) components?

<img width="653" alt="image" src="https://user-images.githubusercontent.com/11774799/217296676-100ed30c-d189-4c87-8a5a-ea499f5a2498.png">

**Comment by loreina** [comment]
Hi @brandongregoryscott,

Unfortunately, it's not possible to remove the border. You're right that this use case calls for using the [Modal Dialog Primitive component](https://paste.twilio.design/primitives/modal-dialog-primitive). You could also try using `spaceNegative` for negative margins to achieve the effect, but it might be safer to use the primitive. Hope this helps!

**Comment by brannytempy** [comment, marked as answer]
@brandongregoryscott I followed up in Slack, but sharing here for visibility as well. I think we should try and keep as close to Paste's `Modal` as possible. I think we can simply shift our art direction in those images to be a light theme so the white border is no longer visible.
